src/components/Buidlhub/BuidlhubClient.ts

```diff
@@ -37,7 +37,8 @@ export default class BuidlhubClient implements IBuidlhubClient {
     }
   }
 
-  async _post(body: any) {
+  //eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed? I suspect it is because you didn't figure out a better alternative to any. I think there likely is a better way and would prefer that we find it.